libraries/AP_NavEKF3/AP_NavEKF3_Outputs.cpp

```diff
@@ -45,6 +45,20 @@ float NavEKF3_core::errorScore() const
         score = MAX(score, 0.5f * (velTestRatio + posTestRatio));
         // Check altimeter fusion performance
         score = MAX(score, hgtTestRatio);
+        // Check airspeed fusion performance - only when we are using at least 2 airspeed sensors so we can switch lanes with 
+        // a better one. This only comes into effect for a forward flight vehicle. A damping factor of 0.3 is added to keep the
+        // EKF less sensitive to innovations arising due events like strong gusts of wind, thus, prevent reporting high error scores
+        if (assume_zero_sideslip()) {
+            const auto *arsp = AP::airspeed();
+            if (arsp->get_num_sensors() >= 2 && frontend->_affinity & EKF_AFFINITY_ARSP) {
```

Comment:
use () around bitops

Reply:
we may want to scale the airspeed contribution with 1/sqrt(airspeed) to prevent too much contribution at low speeds?